src/renderer/components/app.tsx

```diff
@@ -92,24 +92,20 @@ export class App extends React.Component {
         broadcastMessage(`cluster-warning-event-count:${getHostedCluster().id}`, count);
       }),
 
-      reaction(() => clusterPageMenuRegistry.getRootItems(), (rootItems) => {
-        this.generateExtensionTabLayoutRoutes(rootItems);
-      }, {
-        fireImmediately: true
+      reaction(getHostedCluster, () => {
```

Comment:
This looks weird. What it the intention here? 
I suppose we can setup `startUrl` without any reactions since it should be available when app is renders first time.

Reply:
Fixed